ibis/pandas/execution.py

```diff
@@ -239,6 +245,17 @@ def execute_any_all_series(op, data, scope=None):
     return getattr(data, type(op).__name__.lower())()
 
 
+@execute_node.register(ops.CountDistinct, pd.Series)
+def execute_count_distinct_series(op, data, scope=None):
+    # TODO(phillipc): Does count distinct have a mask?
+    return data.nunique()
+
+
+@execute_node.register(ops.Count, pd.DataFrame, type(None))
+def execute_count_frame(op, data, _, scope=None):
+    return len(data)
```

Comment:
shouldn't this be ``data.count()`` this is a 1 dim reduction no, e.g. return a pd.Series).